test/boost/README.md

```diff
@@ -68,6 +68,42 @@ Boost tests can also be run using test.py - which is a script that provides
 a uniform way to run all tests in scylladb.git - C++ tests, Python tests,
 etc.
 
+## Execution with pytest
+
+To run test with pytest execute 
+```bash
+pytest test/boost
+```
+To execute only one file, provide the path filename
+```bash
+pytest test/boost/aggregate_fcts_test.cc
+```
+Since it's a normal path, autocompletion works in the terminal out of the box.
+
+To provide a specific mode, use the next parameter `--mode dev`,
+if parameter isn't provided pytest tries to use `ninja mode_list` to find out the compiled modes.
+
+Parallel execution is controlled by `pytest-xdist` and the parameter `-n auto`.
+This command starts tests with the number of workers equal to CPU cores.
+The useful command to discover the tests in the file or directory is 
+```bash
+pytest --collect-only -q --mode dev test/boost/aggregate_fcts_test.cc
+```
+That will return all test functions in the file.
+To execute only one function from the test, you can invoke the output from the previous command.
+However, suffix for mode should be skipped.
+For example,
+output shows in the terminal something like this `test/boost/aggregate_fcts_test.cc::test_aggregate_avg.dev`.
+So to execute this specific test function, please use the next command 
+```bash
+pytest --mode dev test/boost/aggregate_fcts_test.cc::test_aggregate_avg
+```
+There's a parameter `--repeat` that used to repeat the test case several times in the same way as test.py did.
```

Comment:
I asked this before - what implements this "--repeat" option?

Reply:
The repeat option implements repeats of the test. If repeat is two, the test will be executed twice. If repeat is three, the test will be executed three times. If repeat is `n`, the test will be executed `n` times.

Reply:
But my question is where is this implemented? I didn't see the implementation in this PR. Where is it? (or maybe I just missed it?)

Reply:
test/pylib/cpp/common_cpp_conftest.py::pytest_collection_modifyitems

Reply:
So "--repeat" is just for C++ tests? Why? Don't we need it for all tests? Or maybe that's just a start and later you plan to move it to a different location?

Reply:
Exactly, as for now this is only for C++ tests. Just to support basic functionality as test.py and for better integration with it. It's just a start, all pytest will have this option. I hope I'll find a way to use fixtures in C++ tests, so we will be able to use pytest-repeat module.